packages/docs/content/components/radio.mdx

```diff
@@ -86,20 +86,59 @@ This component also makes use of form field styles, which can be customized by t
 
-### Accessibility
+### Accessibility testing
 
-- Surround a related set of choices with a `<fieldset>`. The `<legend>` provides context for the grouping. Do not use `fieldset` and `legend` for a single checkbox.
-- Some screen readers read the `legend` text for each `fieldset`, so it should be brief and descriptive.
+#### General observations
+
+- No other interactive elements (such as links or buttons) should be included inside the label or hint text.
+- A group of radio buttons should be wrapped in a `<fieldset>` that includes a `<legend>` element within.
+  - The `<legend>` provides context for the grouping, much like a label.
+  - Do not use `fieldset` and `legend` for a single radio button element.
 - Each input should have a semantic `id` attribute, and its corresponding `label` should have the same value in its `for` attribute.
-- The custom checkboxes and radio buttons here are accessible to screen readers because the default fields are moved off-screen.
-- `checkedChildren` will be announced to screen readers when they are exposed. They have been tested with the following devices:
-  - Windows 10 + Internet Explorer 11 + JAWS screen reader
-  - Windows 10 + Chrome + JAWS
-  - Windows10 - Firefox + NVDA
-    - NVDA reads out the `<select>` label and every `<option>` value
-  - MacOS Mojave + Safari + VoiceOver
-
-### Related patterns
+
+#### Keyboard testing
+
+- Each group of radio buttons can be accessed by keyboard using the tab key.
+- Each radio button within a group can be accessed by keyboard using the arrow keys.
+- Arrow keys are used to make a selection.
+- There is a visible focus state.
+- The change in state is visible.
+
+If there are `checkedChildren` being used and...
+
+- ...short conditional content, such as an alert, is present, users can tab to any interactive elements within the conditional content.
+- ...lengthier conditional content is present, users should most likely be on the first legend or label element of the conditional content. They can then tab to any interactive elements within the conditional content.
+
+#### Screen reader testing
+
+**For desktop users**
+
+- Some screen readers read the `legend` text for each form element within, so it should be brief and descriptive.
+- In a radio group, I should hear the question or title related to them via a legend inside a fieldset.
+- Each label should be announced for every radio button.
+- The radio button element should be identified with a role of radio.
+- If hints or errors are present, they are read after the label.
+- When the radio button is selected, that state is read out
+
+If there are `checkedChildren` being used and...
+
+- ...short conditional content, such as an alert, is present, users will hear the child elements read out after the radio is selected because of the aria-live being used.
+- ...lengthier conditional content is present, users will not hear the child elements being read as they are displayed.
```

Comment:
👍 Should we carry over this stategy to checkbox for now until we can get a final a11y review?